pkg/services/dashboards/models.go

```diff
@@ -26,13 +26,14 @@ const (
 
 // Dashboard model
 type Dashboard struct {
-	ID       int64  `xorm:"pk autoincr 'id'"`
-	UID      string `xorm:"uid"`
-	Slug     string
-	OrgID    int64 `xorm:"org_id"`
-	GnetID   int64 `xorm:"gnet_id"`
-	Version  int
-	PluginID string `xorm:"plugin_id"`
+	ID         int64  `xorm:"pk autoincr 'id'"`
+	UID        string `xorm:"uid"`
+	Slug       string
+	OrgID      int64 `xorm:"org_id"`
+	GnetID     int64 `xorm:"gnet_id"`
+	Version    int
+	PluginID   string `xorm:"plugin_id"`
+	APIVersion string `xorm:"api_version"`
```

Comment:
strangely triangular 

![image](https://github.com/user-attachments/assets/e74e6abe-e2ff-47e6-aa61-25b6c9d89cc2)

Reply:
I'm gonna steal the Declaration of Independence...